modules/microshift-multus-intro.adoc

```diff
@@ -6,21 +6,49 @@
 = Additional networks in {microshift-short}
 
-During cluster installation, the _default_ pod network is configured with default values unless you customize the configuration. The default network handles all ordinary network traffic for the cluster. Using the Multus CNI plugin, you can add additional interfaces to pods from other networks. This gives you flexibility when you configure pods that deliver network functionality, such as switching or routing.
+During cluster installation, the _default_ pod network is configured with default values unless you customize the configuration. The default network handles all ordinary network traffic for the cluster. Using the {microshift-short}Multus CNI plugin, you can add additional interfaces to pods from other networks. This gives you flexibility when you configure pods that deliver network functionality, such as switching or routing.
 
-[id="additional-network-use-cases"]
+[id="microshift-supported-additional-networks_{context}"]
+== Supported additional networks for network isolation
+The following additional networks are supported in {microshift-short} {product-version}:
+
+* Bridge
+* IPVLAN
+* MACVLAN
+
+[NOTE]
+====
+Setting network policies for additional networks is not supported.
+====
+
+[id="microshift-additional-network-use-cases_{context}"]
 == Use case: Additional networks for network isolation
 
-You can use an additional network in situations where network isolation is needed, including control plane and data plane separation. You can create additional interfaces for pods to connect to that network in addition to a default. For example, you can configure an additional interface if you want pods to access a network on the host and also communicate with devices deployed to the edge that are on an isolated operator network or are periodically disconnected.
+You can use an additional network in situations where network isolation is needed, including control plane and data plane separation. You can create additional interfaces for pods to connect to that network in addition to a default. For example, you can configure an additional interface if you want pods to access a network on the host and also communicate with devices deployed to the edge. These edge devices might be on an isolated operator network or are periodically disconnected.
 
 Isolating network traffic is useful for the following performance and security reasons:
 
 Performance:: You can send traffic on two different planes to manage the amount of traffic on each plane.
 Security:: You can send sensitive traffic onto a network plane that is managed specifically for security considerations, and you can separate private data that must not be shared between tenants or customers.
 
-[id="additional-network-how-implemented"]
+[NOTE]
+====
+The {microshift-short} service deploys the Multus CNI plugin at start up, so a host restart might be required depending on when the `microshift-multus` RPM package is installed.
```

Comment:
As mentioned in other PR: we want to reboot the host, so all containers are re-created with Multus (even if it only delegates to ovn-k)